components/script/dom/texttracklist.rs

```diff
@@ -125,7 +124,7 @@ impl TextTrackListMethods for TextTrackList {
         self.dom_tracks
             .borrow()
             .iter()
-            .filter(|track| track.id() == &id_str)
+            .filter(|track| track.id() == id_str)
             .next()
```

Comment:
Is there a reason why this particular `.filter(_).next()` chain is not replaced with `.find`?

Reply:
It seems that I overlooked this piece of code, I apologise for the mistake.
It has been fixed now!